sdk/identity/Azure.Identity/api/Azure.Identity.netstandard2.0.cs

```diff
@@ -136,6 +135,7 @@ public partial class DeviceCodeCredentialOptions : Azure.Identity.TokenCredentia
         public bool AllowUnencryptedCache { get { throw null; } set { } }
         public Azure.Identity.AuthenticationRecord AuthenticationRecord { get { throw null; } set { } }
         public string ClientId { get { throw null; } set { } }
+        public System.Func<Azure.Identity.DeviceCodeInfo, System.Threading.CancellationToken, System.Threading.Tasks.Task> DeviceCodeCallback { get { throw null; } set { } }
```

Comment:
Nit - should this have `Async` in the name?  `DeviceCodeCallbackAsync` doesn't sound great though.  Is there another name we could use like `DeviceCodeAcquiredAsync` or something?  It kind of mixes the naming patterns for events with properties, but we're trying to use this property like an event.